src/main/kotlin/no/nav/faktureringskomponenten/service/FakturaGenerator.kt

```diff
@@ -101,14 +101,33 @@ class FakturaGenerator (
             fakturaLinje = fakturaLinjer.sortedByDescending { it.periodeFra })
     }
 
+    private fun erNesteKvartalOgKvartalsbestillingHarKjørt(
+        fakturaStartDato: LocalDate,
+        dagensDato: LocalDate,
+    ): Boolean {
+        val erNesteKvartal = dagensDato < fakturaStartDato && dagensDato[IsoFields.QUARTER_OF_YEAR]
+            .plus(1) % 4 == fakturaStartDato[IsoFields.QUARTER_OF_YEAR] % 4
+        val sisteMånedIDagensKvartal = dagensDato.month.firstMonthOfQuarter().plus(2)
+        val kvartalsBestilingHarKjørt =
+            dagensDato > LocalDate.now().withMonth(sisteMånedIDagensKvartal.value).withDayOfMonth(19)
```

Comment:
Samme logikk her flere steder. Kanskje trekke ut i funksjon med godt navn?

Reply:
dagensDato.month.firstMonthOfQuarter() denne tenker du? 
Er det så mye som er verdt å hente ut her? Måtte evt være til en dato klasse da tenker jeg, for å kunne bruke det flere steder(?)